packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.html

```diff
@@ -0,0 +1,65 @@
+<template>
+  <fieldset class={formClass}>
+    <legend class="slds-form-element__legend slds-form-element__label slds-text-heading_small">
+      {label}
+    </legend>
+    <div class="slds-form-element__control" aria-describedby="case-classification-error-message">
+      <div class="row">
+        <template if:true={loading}>
+          <div class="loading-holder slds-var-m-top_x-small">
+            <lightning-spinner alternative-text="Loading" size="small"></lightning-spinner>
+          </div>
+        </template>
+        <template if:false={loading}>
+          <template for:each={suggestions} for:item="option">
+            <div key={option.value}
+              class="case-classification-suggestion slds-visual-picker slds-visual-picker_medium slds-m-left_none slds-var-m-right_medium slds-var-m-top_x-small">
+              <input type="radio" data-suggestion-id={option.id} onchange={handleSelectClassification} id={option.value} value={option.value} name='options' />
```

Comment:
```suggestion
              <input type="radio" data-suggestion-id={option.id} onchange={handleSelectClassification} id={option.value} value={option.value} name='suggestions' />
```

Reply:
The suggestions and the inline options are both displayed using radio inputs and that's why I used the name value `options`, to have all the radio inputs with the same name. otherwise if the suggestions are named `suggestions` and the inline options are named `options` the user will have the possibility to choose more than one option.